test/files/neg/t5389.scala

```diff
@@ -0,0 +1,4 @@
+
+import ne.scala
```

Comment:
So @jducoeur's paying you under the table after all, hmm.

Reply:
I've owned the domain for less than 24 hours, and I'm already a world power!